packages/expo-cli/src/credentials/views/Select.ts

```diff
@@ -135,7 +134,7 @@ export class SelectAndroidExperience implements IView {
     this.askAboutProjectMode = false;
 
     if (this.androidCredentials.length === 0) {
-      this.androidCredentials = get(await ctx.api.getAsync('credentials/android'), 'credentials');
+      this.androidCredentials = (await ctx.api.getAsync('credentials/android')).credentials;
```

Comment:
```suggestion
      this.androidCredentials = (await ctx.api.getAsync('credentials/android'))?.credentials;
```

Maybe? Just guessing from the `get` command.

Reply:
Good call. The return value was `any` so it was masking the error 👿

Reply:
Although I'm still not sure if it's necessary... might just push the error down the line ...